app/classifier/tasks/survey/choice.cjsx

```diff
@@ -44,15 +44,19 @@ ImageFlipper = React.createClass
 module.exports = React.createClass
   displayName: 'Choice'
 
+  propTypes:
+    annotation: React.PropTypes.object.isRequired
```

Comment:
```javascript
annotation: React.PropTypes.shape({
  answers: React.PropTypes.object,
  choice: React.PropTypes.string
})
```
shouldn't generate any warnings from eslint. I'm not 100% sure if that's exactly what a survey task annotation looks like, but I think it's right.